spring-batch-core/src/main/java/org/springframework/batch/core/explore/JobExplorer.java

```diff
@@ -49,6 +49,16 @@
 	 */
 	List<JobInstance> getJobInstances(String jobName, int start, int count);
 
+	/**
+	 * Find the last job instance for the given job.
+	 * @param jobName name of the job
+	 * @return the last job instance if any or null otherwise
+	 */
```

Comment:
`@since`
Also, what criteria do we use for "last"?  Last started?  Last completed?  Last by id?

Reply:
It is the last by Id. I updated the Javadoc accordingly.